client/cli/src/error.rs

```diff
@@ -18,61 +18,87 @@
 
 //! Initialization errors.
 
-
+use sp_core::crypto;
 
 /// Result type alias for the CLI.
 pub type Result<T> = std::result::Result<T, Error>;
 
 /// Error type for the CLI.
-#[derive(Debug, derive_more::Display, derive_more::From)]
+#[derive(Debug, thiserror::Error)]
```

Comment:
So why do we need this? :D

Reply:
Technically we do not _need_ it.
For consistency, for convenience, for not having to implement `std::error::Error` manually and update it manually it seemed reasonable to convert to `thiserror`.

Reply:
@bkchr is that a strong opposition or just a `I don't want random garbage in there` call?

Reply:
> For consistency, for convenience, for not having to implement std::error::Error manually and update it manually it seemed reasonable to convert to thiserror.

Thought it was doing more. So, it should be fine.

Reply:
That's the beauty, it literally removes the chores.